.travis.yml

```diff
@@ -0,0 +1,20 @@
+env:
+  - PROTOBUF_VERSION=2.6.1
```

Comment:
Can we have protobuf_version 2.3, 2.4.1, 2.5 as well?
Or at least 2.5, since that is what we are still using.